src/components/Contact.css

```
@@ -0,0 +1,189 @@
+.contact-container {
+    position: relative;
+    top: 0;
+    left: 0;
+}
+
+.contact-container img {
+    width: 100%;
+    position: relative;
+    top: 0;
+    left: 0;
+}
+
+.bg-lg {
```

make this all into one css statement not three different ones.
```suggestion
.bg-lg,
.bg-md,
.bg-sm {
```